app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] [optiional] @Zeph180 great job implementing all changes requested, however, I would recommend removing the `foreign key` part, this is usually set in a has-many part of the relationship, you don't need it here.